cinema/views.py

```diff
@@ -0,0 +1,55 @@
+from django.shortcuts import get_object_or_404
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movie = Movie.objects.all()
+        serializer = MovieSerializer(movie, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    if request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid(raise_exception=True):
+            serializer.save()
+            return Response(
+                serializer.errors,
+                status=status.HTTP_201_CREATED
+            )
+        return Response(
+            serializer.errors,
+            status=status.HTTP_400_BAD_REQUEST
+        )
```

Comment:
u don't need to check it using "if"
```suggestion
        serializer.is_valid(raise_exception=True)
        serializer.save()
        return Response(
            serializer.errors,
            status=status.HTTP_201_CREATED
            )
```
u don't need to send errors in response also